ceph_installer/controllers/mon.py

```diff
@@ -53,5 +53,28 @@ def configure(self):
 
     # we need validation here
     @configure.when(method='POST', template='json')
+    @validate(schemas.mon_configure_schema, handler="/errors/schema")
     def configure_post(self):
-        return {}
+        hosts = [request.json.get('host')]
```

Comment:
since we are past validation, it should be safe to do `request.json['host']` here, added bonus that if for *whatever reason* we are not enforcing the need for `host` then it blows up